man/as.networkDynamic.Rd

```diff
@@ -11,54 +11,81 @@
 \alias{as.networkDynamic}
 \alias{as.networkDynamic.network}
 \alias{as.networkDynamic.networkDynamic}
+\alias{as.networkDynamic.networkLite}
 
 \title{
 as.networkDynamic
 }
+
 \description{
-Functions with these names formerly provided S3-dispatched functions for converting between various dynamic network formats which have been DEPRECATED. Use \code{\link{networkDynamic}} instead. Current application only sets the networkDynamic class on objects. 
+The \code{as.networkDynamic} generic provides a very basic conversion to 
+\code{networkDynamic} from other network types. It is generally recommended 
+to use the \code{\link{networkDynamic}} function instead of 
+\code{as.networkDynamic}, because \code{\link{networkDynamic}} provides a 
+number of additional features for handling temporal information.
 }
 
-
-
 \usage{
                  
 \method{as.networkDynamic}{networkDynamic}(object,...)
 
 \method{as.networkDynamic}{network}(object,...)
 
+\method{as.networkDynamic}{networkLite}(object,...)
 }
 %- maybe also 'usage' for other objects documented here.
 \arguments{
-  \item{object}{a \code{network} or \code{networkDynamic} object}
+  \item{object}{a \code{network}, \code{networkLite}, or \code{networkDynamic} object}
 
   \item{\dots}{ignored}
 
 }
 \details{
- \code{as.networkDynamic.network} converts a network object into networkDynamic object by adding a \code{networkDynamic} class.
-}
+ \code{as.networkDynamic.network} converts a \code{network} object into 
+ a \code{networkDynamic} object by adding a \code{networkDynamic} class.
+ 
+ \code{as.networkDynamic.networkLite} converts a \code{networkLite} object into 
+ a \code{networkDynamic} object by first converting it to a \code{network} 
+ object and then adding a \code{networkDynamic} class. (\code{networkLite}s are
+ a backend data structure used in some \code{EpiModel} and \code{statnet}
+ packages, and are not typically handled directly by the end user.)
```

Comment:
.. allowing for more performant updates, at the cost of flexibility in expressing attributes  (limited to vectors only?)

Reply:
addressed in 9fc7480f665afdc9245d1c2ba808bb99d14c89b7

note that vertex and edge attributes are no longer required to be atomic